CHANGELOG.md

```diff
@@ -39,22 +44,72 @@ reason")`.
   extract probability metrics from `TranscriptionFrame` objects for Whisper-based,
   OpenAI GPT-4o-transcribe, and Deepgram STT services respectively.
 
-- Added `LLMSwitcher.register_direct_function()`. It works much like
-  `LLMSwitcher.register_function()` in that it's a shorthand for registering
-  functions on all LLMs in the switcher, but for direct functions.
-
-- Added `LLMSwitcher.register_direct_function()`. It works much like
-  `LLMSwitcher.register_function()` in that it's a shorthand for registering
-  a function on all LLMs in the switcher, except this new method takes a direct
-  function (a `FunctionSchema`-less function).
-
-- Added `MCPClient.get_tools_schema()` and `MCPClient.register_tools_schema()`
-  as a two-step alternative to `MCPClient.register_tools()`, to allow users to
-  pass MCP tools to, say, `GeminiLiveLLMService` (as well as other
-  speech-to-speech services) in the constructor.
-
-- Added support for passing in an `LLMSwicher` to `MCPClient.register_tools()`
-  (as well as the new `MCPClient.register_tools_schema()`).
+- Introduced new `AggregatedLLMTextFrame` type to support representing effective llm
+  output whether or not it is processed by the TTS. This new frame type includes the
+  field `aggregated_by` to represent the conceptual format by which the given text
+  is aggregated. `TTSTextFrame`s now inherit from `AggregatedLLMTextFrame`.
+
+- New `bot-output` RTVI message to represent what the bot actually "says".
+  - The `RTVIObserver` now emits `bot-output` messages based off the new `AggregatedLLMTextFrame`s
+    (`bot-tts-text` and `bot-llm-text` are still supported and generated, but `bot-transcript` is
+    now deprecated in lieu of this new, more thorough, message).
+  - The new `RTVIBotOutputMessage` includes the fields:
+    - `spoken`: A boolean indicating whether the text was spoken by TTS
+    - `aggregated_by`: A string representing how the text was aggregated ("sentence", "word",
```

Comment:
Actually the literal "custom" or "<my custom aggregation>", like in `Aggregation.type`? I know I can read on to get the answer, but just making a note here.

Reply:
Ah, got it. It's the latter. So maybe here use "my custom aggregation", like you do elsewhere to indicate that the string is the developer-provided aggregation type string